src/components/OnboardingFlow.js

```diff
@@ -138,18 +95,21 @@ const OnboardingFlow = ({ onComplete, showMissionSlide }) => {
       onComplete()
     }
   }
+
   return (
     <Card className={classes.card}>
       <CardContent className={classes.cardContent}>
         <img
-          alt="cute cat"
+          alt="onboarding"
           className={classes.onboardingImage}
           src={onboardingStepInfo.imageSrc}
         />
-        <Typography variant="h5" className={classes.cardTitle}>
+        <Markdown className={classes.cardTitle}>
```

Comment:
`Markdown` doesn't actually use any other props, so I think this class is doing nothing. Just use a div?